README.md

```diff
@@ -1,15 +1,36 @@
 # dbt-coverage
 
-<a href="https://github.com/slidoapp/dbt-coverage/blob/main/LICENSE.md"><img alt="License: MIT" src="https://img.shields.io/github/license/slidoapp/dbt-coverage"></a>
 <a href="https://pypi.org/project/dbt-coverage/"><img alt="PyPI" src="https://img.shields.io/pypi/v/dbt-coverage"></a>
 <a href="https://pepy.tech/project/dbt-coverage"><img alt="Downloads" src="https://pepy.tech/badge/dbt-coverage"></a>
 ![GitHub last commit](https://img.shields.io/github/last-commit/slidoapp/dbt-coverage)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/dbt-coverage)
 ![PyPI - Format](https://img.shields.io/pypi/format/dbt-coverage)
+<a href="https://github.com/slidoapp/dbt-coverage/blob/main/LICENSE.md"><img alt="License: MIT" src="https://img.shields.io/github/license/slidoapp/dbt-coverage"></a>
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fslidoapp%2Fdbt-coverage.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fslidoapp%2Fdbt-coverage?ref=badge_shield)
+
+
+_One-stop-shop for docs and test coverage of [`dbt`](https://github.com/dbt-labs/dbt) projects._
+
+## Why do I need something like this?
+
+_**[`dbt-coverage`](https://github.com/slidoapp/dbt-coverage) is to [`dbt`](https://github.com/dbt-labs/dbt) what [`coverage.py`](https://github.com/nedbat/coveragepy) and [`interrogate`](https://interrogate.readthedocs.io/en/latest/) are to Python.**_
+
+It is a single CLI tool which checks your `dbt` project for missing docstrings and tests.
+
+Keeping tests and documentation close to the actual SQL code that generates the final model is one of the best design choices of `dbt`. It ensures tests are actually used and documentation is actually useful. But how do you make adding those a habit in your [`dbt`](https://github.com/dbt-labs/dbt) project?
 
+That is exactly where `dbt-coverage` comes in. It will
+
+- Give you a better sense of how well is your project documented and covered with tests;
+- Help your CI/CD pipeline make sure new changes include documentation and tests;
+- Let you quickly asses the documentatio and tests of a new `dbt` project you get your hands on.
```

Comment:
@mrshu, I'm not sure I got this correctly. Does it mean that thanks to `dbt-coverage` we can quickly know how well a new project is documented and tested?

Reply:
@sweco yes, that's exactly what I've had in mind 🙂